tsconfig.json

```diff
@@ -9,12 +9,12 @@
 		"isolatedModules": true,
 		"target": "ESNext",
 		"module": "ESNext",
-		"moduleResolution": "NodeNext",
+		"moduleResolution": "Bundler",
```

Comment:
Oh that is cool! I didnt see they had a new [moduleResolution `bundler`](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#moduleresolution-bundler)